api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -95,6 +95,17 @@
      */
     ConfigBuilder withConverters(Converter<?>... converters);
 
+    /**
+     * Determines whether variable expansion is enabled.
+     *
+     * This enables variable expansion globally for any `Config` built by this builder.
+     * However, it is possible to override this by using
```

Comment:
The word "override" may be misleading. This value should have no impact on the ConfigAccessorBuilder value. We should add a reference that this only defines the behaviour of Config.getValue and Config.getOptionalValue.

Reply:
+1, I'll rephrase